app/src/main/java/com/medmapper/v33001/dto/MedicineLog.kt

```diff
@@ -1,12 +1,13 @@
 package com.medmapper.v33001.dto
 import com.medmapper.v33001.dto.Medicine
 import com.medmapper.v33001.dto.User
+import java.time.LocalDateTime
 
 //Log for medication adherence
 
 data class MedicineLog (
     val id: String = "",
     val medicationId: String = "",
     val uid: String = "",
-    val time: Long = 0
-)
\ No newline at end of file
+    val time: LocalDateTime = LocalDateTime.now()
+)
```

Comment:
Using a LocalDateTime instead of a Long for the time property. This will make it easier to perform calculations on the date and time.

Source: https://docs.oracle.com/javase/8/docs/api/java/time/LocalDateTime.html